addons/stock/models/stock_warehouse.py

```diff
@@ -346,6 +346,9 @@ def _create_or_update_sequences_and_picking_types(self):
             else:
                 data[picking_type].update(create_data[picking_type])
                 sequence = IrSequenceSudo.create(sequence_data[picking_type])
+                existing_sequence = IrSequenceSudo.search_count([('company_id', '=', sequence.company_id.id), ('name', '=', sequence.name)], limit=1)
+                if existing_sequence:
+                    sequence.name = _("%s (copy)(%s)") % (sequence.name, str(sequence.id))
```

Comment:
When there are multiple placeholders in a translatable string, it's better to name them in case they are moved during the translation (and to give a bit more of context for translators, not really useful in this case but can be really useful for other cases)
```suggestion
                    sequence.name = _("%(name)s (copy)(%(id)s)", name=sequence.name, id=str(sequence.id))
```

Reply:
thanks for the review changes has been done.